examples/client/client/tasks.nim

```diff
@@ -92,6 +93,17 @@ proc statusContext*(arg: ContextArg) {.async, gcsafe, nimcall,
   # re/set threadvars that don't persist across waku dis/connect
   resetContext()
 
+proc updatePrices() {.async.} =
+  while statusState == StatusState.loggedin:
+    await sleepAsync(5000)
+    if statusState == StatusState.loggedout:
```

Comment:
```suggestion
    if statusState != StatusState.loggedin:
```

Should help avoid ambiguity re: `StatusState.loggingin` and `StatusState.loggingout`.

Reply:
Please see above comment.